tests/lustre/typechecking/shouldFail/inlined_contract_02.lus

```diff
@@ -0,0 +1,10 @@
+
+node top(x: int) returns (y: int);
+(*@contract
+  guarantee y >= l; -- error: Unbound identifier l
```

Comment:
Local variables should not be visible in a contract.

Reply:
fixed.